LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 LMC s.r.o.
+Copyright (c) 2021 Alma Career Czechia s.r.o.
```

Comment:
I would prefer this

```suggestion
Copyright (c) 2021 LMC s.r.o.
Copyright (c) 2024 Alma Career Czechia s.r.o.
```

because as of 2021, there was no Alma Career Czechia. What do you think?

Reply:
I have done a little bit of research here and here it is what can we apply:
- `Copyright (c) 2024 Alma Career Czechia.` - indicates a current holder of the document in the year, here I see the problem with the year that indicates the start of the holding
- `Copyright © 2021-2024 Alma Career Czechia.` - indicates the range of holding the rights to the document, but again there is an end year that needs to be updated
- `Copyright © 2021 Alma Career Czechia.` - indicates the start of the year from which the rights to the document apply, for the best solution
- `Copyright © 2021-2024 Alma Career Czechia, formerly LMC s.r.o.` - indicates the start and the former holder name, but again there is an end year

Reply:
```
Copyright © 2021 Alma Career Czechia, formerly LMC s.r.o.
```

How about this? No end date and acknowledged the former name